### PR TITLE
NeoFS sidechain governance description

### PR DESCRIPTION
**Pull request by alexvanin**

### CODE REVIEW

File: 02-blockchain/pic/gov-invoke.puml [hunk 1]
```diff
@@ -0,0 +1,26 @@
+@startuml
+
+participant "IR Ⰰ" as ir1
+participant "IR 2" as ir2
```
Comment (alexvanin) [inline 02-blockchain/pic/gov-invoke.puml:4]: Name it `participant "IR Ⰱ" as ir2`

File: 02-blockchain/01-governance.md [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+if the main net committee will control list of inner ring nodes, then it will
+control sidechain as well.
+
+The standard scheme for updating Iinner Ring is to add keys to the candidate
+list with some deposit, and then inner ring nodes confirm list update.
+
+![InnerRing list update](pic/gov-irup)
+
+The committee should be able to change the list without accumulating signatures
+of Inner Ring nodes. However, the new list must still hold "at least 2 \ 3 + 1
```
Comment (alexvanin) [inline 02-blockchain/01-governance.md:87]: Either use back ticks for formula or add escape symbol `2 \\ 3 + 1`

**Review by realloc — DISMISSED** [review]

File: 02-blockchain/01-governance.md [hunk 3]
```diff
@@ -0,0 +1,97 @@
+## NeoFS Sidechain Governance
+
+NeoFS uses the sidechain as a database to store meta-information about the
+network: network map, audit results, containers, key mappings and network
+settings. The sidechain works on the same principles as the main net: there are
+no free transactions, and the committee chooses the consensus nodes, etc. To
+operate with the sidechain we need to solve the following problems:
+
+* How the mainnet committee can control Inner Ring nodes and the consensus nodes
+  of the sidechain,
```
Comment (alexvanin) [inline 02-blockchain/01-governance.md:10]: If there are separate sentences in bullet list, then end it with question mark:
```
* How the mainnet committee can control Inner Ring nodes and the consensus nodes
  of the sidechain?

* How Storage Nodes and Inner Ring nodes get sidechain \Gls{GAS} to send
  transactions?
```

File: 02-blockchain/01-governance.md [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+
+* How Storage Nodes and Inner Ring nodes get sidechain \Gls{GAS} to send
+  transactions?
+
+NeoFS Governance model solves these problems with seven "Alphabet" sidechain
+contracts.
+
+### Alphabet contracts
+
+Alphabetic contracts are seven contracts deployed in the side-chain, named after
```
Comment (alexvanin) [inline 02-blockchain/01-governance.md:20]: Replace `side-chain` with `sidechain`

File: 02-blockchain/01-governance.md [hunk 5]
```diff
@@ -0,0 +1,97 @@
...
+After receiving \Gls{GAS}, the nodes of the Inner Ring can periodically transfer
+a share to all registered storage nodes and use the received utility token for
+sidechain operations: change epochs, register new containers, save audit
+results, etc.
+
+### Changing sidechain validators
+
+Beforehand, inner ring nodes register new validator keys in the list of
+candidates for the sidechain committee. Then inner ring nodes invoke
+Vote([]keys) method on corresponding alphabetical contracts. Contract votes for
```
Comment (alexvanin) [inline 02-blockchain/01-governance.md:67]: Put `Vote([]keys)` into back ticks `

File: 02-blockchain/01-governance.md [hunk 6]
```diff
@@ -0,0 +1,97 @@
...
+![InnerRing list update](pic/gov-irup)
+
+The committee should be able to change the list without accumulating signatures
+of Inner Ring nodes. However, the new list must still hold "at least $2/3+1$
+former nodes" property.
+
+![InnerRing list update by Mainnet Committee](pic/gov-ircom)
+
+`UpdateInnerRing()` method checks if the sender is a committee and does not
+accumulate the signatures of the inner ring nodes.
```
Comment (alexvanin) [inline 02-blockchain/01-governance.md:97]: `Inner Ring` instead of `inner ring`

File: 02-blockchain/01-governance.md [hunk 7]
```diff
@@ -0,0 +1,97 @@
...
+
+After receiving \Gls{GAS}, the nodes of the Inner Ring can periodically transfer
+a share to all registered storage nodes and use the received utility token for
+sidechain operations: change epochs, register new containers, save audit
+results, etc.
+
+### Changing sidechain validators
+
+Beforehand, inner ring nodes register new validator keys in the list of
+candidates for the sidechain committee. Then inner ring nodes invoke
```
Comment (alexvanin) [inline 02-blockchain/01-governance.md:66]: `Inner Ring` instead of `inner ring` in this line and line above

File: 02-blockchain/01-governance.md [hunk 8]
```diff
@@ -0,0 +1,97 @@
...
+accounts (approximately 14,285,000 for each). By storing \Gls{NEO} on the
+contract accounts, we protect it from unauthorized use by malicious sidechain
+nodes. Contracts do not transfer NEO and use it to vote for sidechain
+\Glspl{validator} and to emit \Gls{GAS}.
+
+### Invocation by InnerRing
+
+Contracts cannot distribute utility token or vote by themselves. To do these
+operations, inner ring nodes invoke alphabetical contract methods. First seven
+inner ring nodes from the inner ring list can invoke corresponding alphabetical
```
Comment (alexvanin) [inline 02-blockchain/01-governance.md:33]: `Inner Ring` instead of `inner ring` in this line and line above

**Review by realloc — APPROVED** [review]

**Pull request merged**